.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "java.compile.nullAnalysis.mode": "automatic"
+  "java.compile.nullAnalysis.mode": "automatic",
+  "svg.preview.background": "white"
```

Comment:
Just note for future, there is no need to add IDE specific files in project since different developers might use different IDEs